src/txn/txn_ckpt.c

```diff
@@ -606,6 +609,9 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, const char *cfg[])
 			    sizeof(timestamp_config),
 			    "read_timestamp=%s", timestamp_buf));
 			txn_cfg[2] = timestamp_config;
+			__wt_verbose(session, WT_VERB_TIMESTAMP,
+			    "Checkpoint requested at stable timestamp: %s",
+			    timestamp_config);
```

Comment:
Minor nit: Why not have this use the `wt_verbose_timestamp` function as well?  It may be less convenient because it is already embedded in a string but the consistency would be nice.